core/src/main/java/io/micronaut/core/io/file/DefaultFileSystemResourceLoader.java

```diff
@@ -129,6 +129,9 @@ private static String normalize(String path) {
         }
         if (path.startsWith("file:")) {
             path = path.substring(5);
+            if (path.startsWith("//")) {
```

Comment:
Why this change? can you add a test?

Reply:
Problem with pathes in windows.... Path like this `//D:/Wok/....` is wrong